docs/estimate.md

```diff
@@ -62,7 +62,7 @@ Tezos.estimate
 
 ```js live noInline wallet
 // import { TezosToolkit } from '@taquito/taquito';
-// const Tezos = new TezosToolkit('https://kathmandunet.api.tez.ie');
+// const Tezos = new TezosToolkit('https://ghostnet.api.tez.ie');
```

Comment:
Some URL mention the old `api.tez.ie` domain. Can you rename all of them to `ghostnet.ecadinfra.com` ?

Reply:
Yes, thanks. Good one!  But it looks like that is the only example. Fixed.

Reply:
is this supposed to stay as api.tez.ie?